### ISSUE TITLE
Application fails silently because of missing runtime, no alert to user

### ISSUE THREAD

**Issue by vasyugan** [flatpak/flatpak#3180]
## Description of the problem I tried several times to run qownnotes, and nothing happened. Then I decided to try from the console:
` flatpak run org.qownnotes.QOwnNotes `

And the resulting output was:

`Fehler: runtime/org.kde.Platform/x86_64/5.13 ist nicht installiert`

So for whatever reason, the required runtime was missing, and that's why the application failed. 

In such a case, there should be some alert by flatpak popping up, alerting the user that the app couldn't be launched because of a missing runtime, offering to them to install it.

**Comment by refi64** [comment]
Normally it shouldn't be possible to install an app without the runtime.

**Comment by mwleeds** [comment]
Yeah it would be good to know how you installed QOwnNotes. Did you use GNOME Software or the CLI?

**Comment by vasyugan** [comment]
> Yeah it would be good to know how you installed QOwnNotes. Did you use GNOME Software or the CLI?

Honestly I don't remember. it's been a while. After I removed qownnotes and reinstalled it from the command line it was file. So  I guess this is irreproducible @mwleeds

**Comment by matthiasclasen** [comment]
it would be good to verify that `flatpak repair` detects and fixes app-without-runtime situations. I guess `flatpak update` should as well, and `flatpak run` should give a clear error.

**Comment by mwleeds** [comment]
Looks like currently `flatpak repair` does not check for a missing runtime, according to https://github.com/flatpak/flatpak/blob/master/app/flatpak-builtins-repair.c#L326
